parkings/models/parking.py

```diff
@@ -80,7 +80,7 @@ class Parking(TimestampedModelMixin, UUIDPrimaryKeyMixin):
     )
     domain = models.ForeignKey(
         EnforcementDomain, on_delete=models.PROTECT,
-        related_name='parkings')
+        related_name='parkings', null=True, blank=True,)
```

Comment:
blank=True is not needed. Or if you leave it here, then add it to 0029 too to avoid changing the field twice in migrations.